setuptools/command/build_ext.py

```diff
@@ -16,15 +16,11 @@
 except ImportError:
     _build_ext = _du_build_ext
 
-try:
-    # Python 2.7 or >=3.2
```

Comment:
This change seems to be moving in the wrong direction, conflicting with the intention of the author to support the newer, preferred module. I don't know what the implications are, but what would prevent someone from making the reverse change again?

I think this behavior needs a unit test in the test suite to capture the desired behavior.

Also, is it possible to allow the newer, preferred module to still be used, but always invoke the `get_config_var`? What is the relationship between `sysconfig._CONFIG_VARS` and `distutils.sysconfig._config_vars`?